web_widget_open_tab/static/src/js/open_tab_widget.esm.js

```diff
@@ -10,21 +8,7 @@ export class OpenTabWidget extends Component {
         ev.stopPropagation();
     }
     _getReference() {
-        var url = window.location.href;
-        var searchParams = new URLSearchParams(url.split("#")[1]);
-        searchParams.set("view_type", "form");
-        searchParams.set("id", this.props.record.data.id);
-        if (
-            !searchParams.has("model") ||
-            searchParams.get("model") !== this.props.record.resModel
-        ) {
-            searchParams.set("model", this.props.record.resModel);
-            searchParams.delete("action");
-        }
-        return url.split("#")[0] + "#" + searchParams.toString();
-    }
-    loadAttrs(ev) {
-        $(ev.target).tooltip();
+        return window.location.pathname + "/" + this.props.record.data.id;
```

Comment:
Use this:

`return `/odoo/${this.props.record.resModel}/${this.props.record.data.id}`;`

Reply:
I've changed it this way, because I also could not find a better solution, but the bread-crumbs and context get lost in the new tab.

Reply:
Might be, but there is no better solution, as we need to use the model if the widget is aplied in a list of a on2many field